static/css/home.css

```diff
@@ -326,18 +326,95 @@ body{
     flex-direction: column;
     align-items: center;
     justify-content: flex-start;
-     /* Enable scrolling if content exceeds screen height */
     border-radius: 30px 30px 0px 0px;
     box-shadow: 0px -3px 5px rgba(0, 0, 0, 0.5);
 }
 
-#nearYouMobileLine {
+#nearYouMobileOverlay.desktop {
```

Comment:
A mobile overlay for desktop? It's a confusing name

Reply:
I renamed the overlay and the top text.
The line is only for mobile, so it still has 'mobile' in the name.
The button is only for desktop, so it still has 'desktop' in the name.